File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task128_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.Random;

public class RandomGenerator {
    public static void main(String[] args) {
        Random random = new Random();
        
        System.out.println("Random Number: " + generateRandomNumber(random));
        System.out.println("Random Token: " + generateRandomToken(random));
    }
    
    public static int generateRandomNumber(Random random) {
        return random.nextInt(100) + 1;
    }
    
    public static String generateRandomToken(Random random) {
        String characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
        StringBuilder token = new StringBuilder();
        for (int i = 0; i < 8; i++) {
            token.append(characters.charAt(random.nextInt(characters.length())));
        }
        return token.toString();
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task128_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <stdlib.h>
#include <time.h>
#include <string.h>

int generateRandomNumber(int minVal, int maxVal) {
    return rand() % (maxVal - minVal + 1) + minVal;
}

void generateRandomToken(char *token, int length) {
    const char characters[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
    int charactersLength = strlen(characters);
    
    for (int i = 0; i < length; i++) {
        token[i] = characters[rand() % charactersLength];
    }
    token[length] = '\\0';
}

int main() {
    srand(time(NULL));
    
    printf("Random number: %d\\n", generateRandomNumber(1, 100));
    
    char token[11];
    generateRandomToken(token, 10);
    printf("Random token: %s\\n", token);
    
    return 0;
}
